README.md

```diff
@@ -22,7 +22,7 @@ $ npm install eslint-visitor-keys
 ## 📖 Usage
 
 ```js
-const evk = require("eslint-visitor-keys")
+import evk from "eslint-visitor-keys"
```

Comment:
We should have both in the docs, like in https://github.com/eslint/espree#usage

Reply:
Sure, added.